control/autopilot/include/autopilot/autopilot_inl.h

```diff
@@ -1147,20 +1222,57 @@ quadrotor_common::ControlCommand AutoPilot::executeTrajectory(
       *trajectory_execution_left_duration = ros::Duration(0.0);
       *trajectories_left_in_queue = 0;
       trajectory_queue_.pop_front();
-      setAutoPilotState(States::HOVER);
-      return base_controller_.run(state_estimate, reference_state_,
+      setAutoPilotStateForced(States::HOVER);
+
+      reference_trajectory_ = quadrotor_common::Trajectory();
+      reference_trajectory_.trajectory_type = 
+        quadrotor_common::Trajectory::TrajectoryType::GENERAL;
+      reference_trajectory_.points.push_back(reference_state_);
+      return base_controller_.run(state_estimate, reference_trajectory_,
                                   base_controller_params_);
     }
     else
     {
       time_start_trajectory_execution_ +=
-          trajectory_queue_.front().points.back().time_from_start;
+        trajectory_queue_.front().points.back().time_from_start;
```

Comment:
I saw a few of those... Detail but this is just a thing because of non equal auto formatting which doesn't hurt but is also not necessary.

Reply:
we use two spaces for tabs... so if I see it I change it :-)